packages/app/src/views/Home.vue

```diff
@@ -1,35 +1,27 @@
 <template>
-  <div class="m-auto">
-    <pre>{{ user }}</pre>
-    <p>Suchefeld</p>
-    <Textfield v-model="text" class="mb-4" icon="search" placeholder="Titelsuche" />
-    <p>Textfield</p>
-    <Textfield v-model="text" class="mb-4" label="Title" placeholder="Titelsuche" />
+  <div id="wrapper" class="mx-auto max-w-lg h-full w-full flex flex-col">
+    <!-- this is a basic template to fill the divs with content. The divs already match roughly the sizes of the figma design -->
+    <div class="h-1/5 bg-blue-300 flex items-center justify-center text-4xl text-gray-800">
```

Comment:
since the _id_ parameter is not needed for anything, maybe we could use that parameter for better readability of the code?
like:
```suggestion
    <div id="header" class="h-1/5 bg-blue-300 flex items-center justify-center text-4xl text-gray-800">
```

we could also use the HTML Layout Tags, which are
`<header>
<section>
<article>
<footer>`

Reply:
sure why not

Reply:
how about using 
`<header>`
`<article> or <details> or <section> or <main>`
`<footer>`
?

Reply:
fixed in e57b4abcd13aa1fca23f2daea03c782cacc55106